daemon/src/engine/server/sv_ccmds.cpp

```diff
@@ -369,6 +385,48 @@ static void SV_Systeminfo_f()
 	Info_Print( Cvar_InfoString( CVAR_SYSTEMINFO, false ) );
 }
 
+class ListMapsCmd: public Cmd::StaticCmd
+{
+public:
+	ListMapsCmd():
+		StaticCmd("listmaps", Cmd::SYSTEM, "Lists all maps available to the server")
+	{}
+
+	void Run(const Cmd::Args&) const OVERRIDE
+	{
+		std::vector<std::string> maps;
+
+		FS::RefreshPaks();
+		std::error_code ignore;
+		for ( const auto& pak : FS::GetAvailablePaks() )
+		{
+			FS::PakPath::LoadPakPrefix(pak, "maps", ignore);
+		}
+
+		for ( const auto& pak : FS::PakPath::ListFiles("maps", ignore) )
+		{
+			if ( Str::IsSuffix(".bsp", pak) )
```

Comment:
I think we need a single place that does something like that (ie: return a list of all map names).
Currently there are similar loops in 3 or 4 places.

Shall I do that as part of this branch?